### PR TITLE
backup, restore, and saved search import API docs

### PR DESCRIPTION
**Pull request by traetox**

### CODE REVIEW

File: api/management.md [hunk 1]
```diff
@@ -25,3 +25,32 @@ An example response when not in distributed mode:
 	"Distributed": false
 }
 ```
+
+
+### Performing a system backup
+
+Admin users may request a system backup which will provide a backup file containing all content related to the state of Gravwell.
+
+A system backup can be used to save user and group accounts, dashboards, kits, query libraries, and even saved searches.  This is essentially everything but data and system configuration.
```
Comment (djfritz) [inline api/management.md:34]: s/system configuration/configuration files (e.g. gravwell.conf)/

File: api/management.md [hunk 2]
```diff
@@ -25,3 +25,32 @@ An example response when not in distributed mode:
...
+
+A backup is obtained by performing a `GET` request on `/api/backup` as an admin user, the API will then return a file download with the backup file.
+
+By default a backup does not contain any saved searches, to include saved searches in the backup append the `savedsearch=true` URL parameter on the GET request.
```
Comment (djfritz) [inline api/management.md:38]:
```suggestion
By default a backup does not contain any saved searches; to include saved searches in the backup append the `savedsearch=true` URL parameter on the GET request.
```

File: api/management.md [hunk 3]
```diff
@@ -25,3 +25,32 @@ An example response when not in distributed mode:
...
+
+The `/api/backup` API may be authenticated using either the JWT authorization token or a cookie.
+
+NOTE: Only a single backup and/or restore may take place at any given time.
+
+### Restoring from a system backup
+
+Admin users may restor the system from a backup archive by performing a `POST` to the `/api/backup` API and uploading a backup file using a multipart form.
```
Comment (djfritz) [inline api/management.md:46]:
```suggestion
Admin users may restore the system from a backup archive by performing a `POST` to the `/api/backup` API and uploading a backup file using a multipart form.
```

**Review by djfritz — CHANGES_REQUESTED** [review]

File: api/management.md [hunk 4]
```diff
@@ -25,3 +25,32 @@ An example response when not in distributed mode:
...
 ```
+
+
+### Performing a system backup
+
+Admin users may request a system backup which will provide a backup file containing all content related to the state of Gravwell.
+
+A system backup can be used to save user and group accounts, dashboards, kits, query libraries, and even saved searches.  This is essentially everything but data and system configuration.
+
+A backup is obtained by performing a `GET` request on `/api/backup` as an admin user, the API will then return a file download with the backup file.
```
Comment (floren) [inline api/management.md:36]:
```suggestion
A backup is obtained by performing a `GET` request on `/api/backup` as an admin user. The API will then return a file download with the backup file.
```

File: api/searchctrl.md [hunk 5]
```diff
@@ -143,3 +143,17 @@ In order to get a list of all searches that exist on the system, an admin user m
 ]
 
 ```
+
+### Importing a saved search archive
+
+An optional download format for a search is an `archive`.  An archive represents a fully self-contained search that can be imported into another Gravwell instance.  The import API accepts the saved search archives as an upload and unpacks the search into the saved search system.  Users can then attach to the search as if it were saved on the local system.
+
+When a search archive is reimported the search becomes owned by the user that imported it, regardless of the user that downloaded it.  An optional `GID` form field may be supplied as part of the import process to also assign a group to the imported search.
```
Comment (floren) [inline api/searchctrl.md:151]:
```suggestion
When a search archive is reimported, the imported search is owned by the user that *imported* it, regardless of which user *downloaded* it.  An optional `GID` form field may be supplied in the import request to share the imported search with a group.
```

**Review by floren — CHANGES_REQUESTED** [review]

**Pull request merged**